cmd/reading/list/list.go

```diff
@@ -40,20 +41,20 @@ const readingTemplate = "Reading ID\tName\tDevice\tOrigin\tValue\tCreated\tModif
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
-		Short: "A list of all device readings",
-		Long:  `Return all device readings.`,
+		Short: "A list of all readings or list of all device readings",
```

Comment:
Diana .. are we not limited to 50 or some limited number as specified by the "limit" flag of readings? "all" implies no limits. how about instead:
"A list of readings across devices or pertaining to a specified device"

Reply:
An issue opened by you https://github.com/edgexfoundry/edgex-go/issues/2560
./edgex-cli reading list (v1/reading endpoint) does not support `limit`
meanwhile ./edgex-cli reading list <device-name> --limit 40 - does support `limit` - In this case the default value if not set is 50

Reply:
v1/reading:
  get:
  description: Return list of all readings, sorted by reading ID.
---------------------------------------------
 /v1/reading/device/{deviceId}/{limit}:
    get:
      description: Return a list of all readings for a given device, sorted
        by the readings modified date.
------------------------------------- 
any suggestion that include in cli helper messages (short/long) these details ?

Reply:
message changed to "A list of readings across devices or pertaining to a specified device"